zulipterminal/ui_tools/views.py

```diff
@@ -411,17 +411,28 @@ def __init__(self, width: int, view: Any) -> None:
                                               header=search_box)
 
     @asynch
-    def update_user_list(self, search_box: Any, new_text: str) -> None:
-        if not self.view.controller.editor_mode:
+    def update_user_list(self, search_box: Any=None,  # type: ignore
+                         new_text: str=None,
+                         user_list: Any=None) -> None:
```

Comment:
I wanted to tighten this up. But it is leading to a cascade of mypy - 'unable to infer type' in further places.